src/components/include/application_manager/application_manager.h

```diff
@@ -585,6 +586,27 @@ class ApplicationManager {
 
   virtual void RemoveAppFromTTSGlobalPropertiesList(const uint32_t app_id) = 0;
 
+  /**
+   * @brief Resets application's global properties to default values
+   * @param container with global properties IDs to reset
```

Comment:
missing param name

Reply:
Fixed in 1c1f20898109fd9781451234e4032784ef97c4f6